index.js

```diff
@@ -44,8 +44,7 @@ queue.process('email', 30, require('./workers/email'));
 queue.process('sendBatchEmails', require('./workers/sendBatchEmails')(queue, db, baseUrl));
 queue.process('sendWeeklyRecapEmails', require('./workers/sendWeeklyRecapEmails')(queue, db, baseUrl));
 queue.process('amazonPayments', require('./workers/amazonPayments')(queue, db));
-
-queue.promote();
```

Comment:
Why? According to kue it's deprecated and no longer needed (also says so during startup)